.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Build and Publish
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - "*"
+    paths-ignore:
+      - "docs/**"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    # TODO: when we have tests
+    # - run: |
+    #   ./scripts/ci/install.sh
+    #   ./scripts/ci/test.sh
+    - name: Publish Docker
```

Comment:
If I was pushed to name one thing to improve, I'd maybe request that the name `Publish Docker` be changed to `Publish to Docker` or `Publish Docker image`